frontend/src/components/ShootDetails/GardenctlCommands.vue

```diff
@@ -15,44 +15,27 @@ limitations under the License.
 -->
 
 <template>
-  <v-card v-if="isAdmin" three-line>
-    <v-toolbar flat dark dense color="cyan darken-2">
-      <v-toolbar-title class="subtitle-1">Gardenctl</v-toolbar-title>
-    </v-toolbar>
-    <v-list>
-      <v-list-item v-for="({ title, value }, index) in commands" :key="title">
-        <v-list-item-icon>
-          <v-icon v-if="index === 0" color="cyan darken-2">mdi-console-line</v-icon>
-        </v-list-item-icon>
-        <v-list-item-content>
-          <v-list-item-subtitle>{{title}}</v-list-item-subtitle>
-          <v-list-item-title>
-            <code-block
-              lang="shell"
-              :content="'$ ' + value.replace(/ --/g, ' \\\n    --')"
-              :show-copy-button="false"
-            ></code-block>
-          </v-list-item-title>
-        </v-list-item-content>
-        <v-list-item-action>
-          <copy-btn :clipboard-text="value"></copy-btn>
-        </v-list-item-action>
-      </v-list-item>
-    </v-list>
-  </v-card>
+  <div>
+    <div v-for="({ title, subtitle, value }, index) in commands" :key="title">
+      <gardenctl-command
```

Comment:
inline code from `gardenctl-command`. To many unnecessary div's .

Reply:
✅